src/app/zap-templates/zcl/zcl.xsd

```diff
@@ -1,19 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-Copyright (c) 2008,2020 Silicon Labs.
```

Comment:
Let's keep the license text, just replace Silicon Labs with
   Copyright (c) 2021 Project CHIP Authors
     All rights reserved.